pkg/cmd/recommend/trigger.go

```diff
@@ -0,0 +1,133 @@
+package recommend
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/gocrane/kubectl-crane/pkg/cmd/options"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/printers"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/klog/v2"
+)
+
+var (
+	triggerRecommendExample = `
+# manually trigger the specified recommendation rule
+%[1]s recommend trigger --name workloads-ntzns -n default
+
+# pre-commit
+%[1]s recommend trigger --name workloads-ntzns -n default --dry-run
+`
```

Comment:
1. "manually trigger the specified recommendation rule " change to "manually trigger the specified recommendation "
2. "workloads-ntzns" looks like a workload name , we need a recommendation name here

Reply:
Done